packages/grpc-bench/grpc-bench.0.2.0/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "grpc benchmarking"
```

Comment:
CI linter is giving the the following warning:
`warning 47: Synopsis should start with a capital and not end with a dot`

I don't think this is a blocker but fixing it will help us by reducing CI noises so we'd appreciate if you can consider about it.

Reply:
Absolutely, this is already in progress in https://github.com/dialohq/ocaml-grpc/pull/43/files#diff-d71eddca4e2da37e56aa8fc421b39f58c01b477013569e977aeb6af77d37f3b7 and will be fixed in the next release.

Reply:
```suggestion
synopsis: "Benchmarking package for gRPC"
```

We can just fix it in this PR as well, so that it doesnt look odd when rendered on the package website.